src/mudlet.cpp

```diff
@@ -3410,6 +3428,12 @@ void mudlet::slot_irc()
     mpIrcClientMap.value(pHost)->show();
 }
 
+
+void mudlet::slot_discord()
+{
+    openWebPage(QStringLiteral("https://discord.gg/kuYvMQ9"));
```

Comment:
The only issue I can see with this is that these Discord URLs can expire/change.   I worry about this being hard-coded and potentially creating versions of the software where the help link is broken.   I don't think this would be an issue if we were using a Vanity URL, as I believe those can only be changed by Discord Support...   

On the other hand, our forum uses something similar, however the "Join Discord" button does make use of the Discord Server Widget information to get the `instant_invite` link which is set in the Discord Server Widget settings page.   
The JSON for our Discord guild/server is available at this address:  `https://discordapp.com/api/guilds/283581582550237184/widget.json`  

If we can afford to make an HTTPS request and parse the JSON for the `instant_invite` attribute I think using that information as well as a fall-back hard-coded value would be a better way to avoid versions of the software with potentially broken Discord links.

Reply:
Sure, I can use the link and in the background download that JSON and update the link when it's available. This'll help future-proof it somewhat.

Reply:
Feedback applied.